docs/02-BUILD_THE_RIGHT_THING.md

```diff
@@ -0,0 +1,37 @@
+## Where do i start?
+So you definitely need a design-system, here are some questions to help guide you in `building the right thing`, before you start trying to `build the thing right`.
+
+1. What are the underlying principles behind our design decisions?
+2. What are the design decisions we value?
+  - [ ] Motion
+  - [ ] Color
+  - [ ] Typography
+  - [ ] Tone of voice
+  - ...
+3. What are the artifacts of our design-system?
+  - [ ] Tokens
+  - [ ] Component Library
+    - [ ] web / native / mobile
+    - [ ] Big screen / little screen / etc
+4. What are the tools Available to me?
+  -  [ ] Languages (css, js, css-in-js, node, swift, kotlin, go?)
+  -  [ ] Team Skills and specialisations
+  -  [ ] Context (?)
+  -  [ ] Sketch
```

Comment:
Moved `sketch`, `storybook` and `docz` here since  from the `Define how / where it is` section as these feel like concerns around the tools we have available to us.